backend/src/test/java/capstone/kpiboard/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,26 @@
+package capstone.kpiboard;
+
+import capstone.kpiboard.exceptions.GlobalExceptionHandler;
+import capstone.kpiboard.exceptions.KpiNotDeletedException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import java.util.Map;
+
+
+@ControllerAdvice
+class GlobalExceptionHandlerTest {
```

Comment:
Ich weiß gar nicht ob ich hierzu einen Unit-Test geschrieben hätte. Wenn du in den Integrations-Tests auch die Fälle testest in denen eine Exception geworfen wird testest du den COntroller Advice ja mit.